packages/core/src/components/button/_button.scss

```diff
@@ -260,6 +261,17 @@ a.pt-button {
   @extend .pt-button:disabled;
 }
 
+.pt-button-loading-spinner {
```

Comment:
I get that. With Adi here, if it's the spinner container, `.pt-button-spinner-container` sounds good